### PR TITLE
Add flame graph support

### PR DESCRIPTION
**Pull request by jnjackins**
This is untested. Help with testing would be appreciated.

### CODE REVIEW

**Comment by Pryz** [comment]
I will help on the testing part :)

File: handler.go [hunk 1]
```diff
@@ -45,8 +45,11 @@ func (h *Handler) ServeHTTP(res http.ResponseWriter, req *http.Request) {
 	case path == "/", path == "/services":
 		h.serveRedirect(res, req, "/services/")
 
-	case path == "/profile":
-		h.serveProfile(res, req)
```
Comment (achille-roussel) [inline handler.go]: Is removing this path expected?

Comment (jnjackins) [reply handler.go]: I'm proposing that /profile becomes /tree, and we add /flame as well.

Comment (jnjackins) [reply handler.go]: "Tree view" being the current graph visualization, even though it's not really a tree. (pprof still calls it a tree)

Comment (achille-roussel) [reply handler.go]: Sounds good :+1:

**Pull request merged**